roles/core/repositories_client/templates/releasever.j2

```diff
@@ -0,0 +1 @@
+{{ equipment_profile.operating_system.repositories_releasever }}
```

Comment:
Don't have infra today to validate this. Did you tested it on both Centos and RHEL?

Reply:
Only CentOS 7 and 8. I can test on RHEL if you want. Behaviour should be the same though.

Reply:
Then perfect. No need more tests.